helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -521,6 +522,15 @@ public void setClusterManagementMode(ClusterManagementModeRequest request) {
     }
   }
 
+  @Override
+  public ClusterManagementMode getClusterManagementMode(String clusterName) {
+    HelixDataAccessor accessor =
```

Comment:
I remember we have getHelixDataAccessor()

Reply:
That's in helix-rest, not in this ZKHelixAdmin implementation. In helix rest, we can use getHelixDataAccessor()